deploy-test.sh

```
@@ -0,0 +1,120 @@
+echo "1. Check ModelsAsService CR:"
+oc get modelsasservice -A
+echo ""
+
+echo "2. Find applications namespace:"
+APPS_NS=$(oc get deployment -A | grep maas-controller | awk '{print $1}' || echo "NOT_FOUND")
+if [ "$APPS_NS" != "NOT_FOUND" ]; then
+  echo "   Applications namespace: ${APPS_NS}"
+  echo ""
+
+  echo "3. Check maas-controller Deployment:"
+  oc get deployment maas-controller -n ${APPS_NS}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)